reactioncommerce/reaction
# Refactor 3570 willopez fix eslint errors

Fixes linting errors caused by enabling several linting rules.

**Testing**
- Run eslint and confirm there are no errors
- Enable shipping
- Enable a payment method
- Place an order
